tools/formatter.py

```diff
@@ -120,6 +120,20 @@ def remove_extras(entry: dict):
 
 	return entry
 
+def remove_duplicate_points(entry: dict):
+	"""
+	Removes points from paths that occur twice after each other
+	"""
+	path: list = entry['path']
+	previous: list = path[0]
+	for i in range(len(path)-1, -1, -1):

```

Comment:
```suggestion
	for i in range(len(path)-1, 0, -1):
```

Oof, man, don't do -1. You will remove the first point, and get this:

![image](https://user-images.githubusercontent.com/11584103/162620048-66384cce-4afe-4f7d-b5d0-95d75c00c642.png)

Reply:
After looking into this, I noticed that I did an error while replacing spaces with tabs, which led to this error.
The range to -1 is correct.
I fixed this bug in the latest commit

Reply:
Aye, seems fixed now.